src/Sulu/Bundle/AdminBundle/Resources/js/components/FolderList/Folder.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React from 'react';
+import Icon from '../Icon';
+import folderStyles from './folder.scss';
+
+const FOLDER_ICON = 'folder-o';
+
+type Props = {
+    /** The id which will be passed as an argument inside the onClick handler */
+    id: string | number,
+    /** The subtext underneath the title */
+    meta: string,
+    title: string,
+    onClick?: (id: string | number) => void,
+};
+
+export default class Folder extends React.PureComponent<Props> {
+    handleClick = () => {
+        if (this.props.onClick) {
+            this.props.onClick(this.props.id);
+        }
+    };
+
+    render() {
+        const {
+            meta,
+            title,
+        } = this.props;
+
+        return (
+            <div
+                className={folderStyles.folder}
+                onClick={this.handleClick}
```

Comment:
That's not very accessible. What about including a `a` or `button` tag somehow?

Reply:
I've just realized that the same is true for our `Icon` component 🙈

Reply:
I am not sure what you mean. I have added `role="button"` and `tabIndex="0"` in the next 2 lines for accessibility. The reason is that you can't use divs inside a button.

Reply:
ah, looks like I have overseen that 🙈